aspnetcore/fundamentals/localization.md

```diff
@@ -98,6 +98,24 @@ DataAnnotations error messages are localized with `IStringLocalizer<T>`. Using t
 
 In ASP.NET Core MVC 1.1.0 and higher, non-validation attributes are localized. ASP.NET Core MVC 1.0 does **not** look up localized strings for non-validation attributes.
 
+<a name=one-resource-string-multiple-classes></a>
+### Using one resource string for multiple classes
+
+The following code shows how to use one resource string for validation attributes with multiple classes:
+
+```
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddMvc()
+        .AddDataAnnotationsLocalization(options => {
+            options.DataAnnotationLocalizerProvider = (type, factory) =>
```

Comment:
@hishamco  You can only highlight when you import a snippet. Inline code can't be highlighted.